cmd/tailscale/cli/up.go

```diff
@@ -609,6 +664,16 @@ func runUp(ctx context.Context, args []string) error {
 	}
 }
 
+func printUpDoneJSON(state, errorString string) {
+	js := &upOutputJSON{BackendState: state, Error: errorString}
+	data, err := json.MarshalIndent(js, "", "  ")
```

Comment:
"\t" for indent? up to you, but fewer bytes on the wire and more semantically an indent (Go bias, admittedly)

Reply:
Done.